packages/commerce-server/src/format-prices-for-product.test.ts

```diff
@@ -113,6 +113,26 @@ test('product with quantity 5 calculatedPrice to have discount applied', async (
   expect(expectedPrice).toBe(product?.calculatedPrice)
 })
 
+test('multiple purchases meeting quantity threshold have bulk discount applied', async () => {
+  const userId = 'user-123'
+  mockCtx.prisma.merchantCoupon.findMany.mockResolvedValue([
+    getMockCoupon('bulk', 0.05),
+  ])
+  mockCtx.prisma.purchase.findMany.mockResolvedValue([
+    getMockExistingBulkPurchase(userId, DEFAULT_PRODUCT_ID, 3),
+  ])
```

Comment:
the test is failing on the `purchase.findMany` call, it seems that it isn't getting properly mocked so it is instead trying to call the DB for which it doesn't have a valid connection string. I'm mocking this call the same way other calls are being mocked though 🤔 

![CleanShot 2022-12-02 at 14 29 56@2x](https://user-images.githubusercontent.com/694063/205380743-0ffc7704-3910-41e3-ba6a-74f422efa88d.png)

Reply:
The issue was that `ctx` (which is mocked in the test) was not being passed to the instance of `getSdk()` that was being used by the code exercised in this test. Passing `ctx` down and replacing `getSdk()` with `getSdk({ ctx })` fixed it.